Foundation/NSGeometry.swift

```diff
@@ -11,6 +11,7 @@
     import Darwin
 #elseif os(Linux)
     import Glibc
+    import CoreFoundation
```

Comment:
Seems reasonable to just import that for everyone; it is likely a bug in the Glibc module map